hyperspy/_components/voigt.py

```diff
@@ -171,58 +145,62 @@ def estimate_parameters(self, signal, E1, E2, only_current=False):
         Examples
         --------
 
-        >>> g = hs.model.components1D.Gaussian()
-        >>> x = np.arange(-10,10, 0.01)
-        >>> data = np.zeros((32,32,2000))
-        >>> data[:] = g.function(x).reshape((1,1,2000))
-        >>> s = hs.signals.Signal1D({'data' : data})
-        >>> s.axes_manager.axes[-1].offset = -10
-        >>> s.axes_manager.axes[-1].scale = 0.01
-        >>> g.estimate_parameters(s, -10,10, False)
+        >>> g = hs.model.components1D.Voigt(legacy=False)
+        >>> x = np.arange(-10, 10, 0.01)
+        >>> data = np.zeros((32, 32, 2000))
+        >>> data[:] = g.function(x).reshape((1, 1, 2000))
+        >>> s = hs.signals.Signal1D(data)
+        >>> s.axes_manager[-1].offset = -10
+        >>> s.axes_manager[-1].scale = 0.01
+        >>> g.estimate_parameters(s, -10, 10, False)
 
         """
         super(Voigt, self)._estimate_parameters(signal)
         axis = signal.axes_manager.signal_axes[0]
+        centre, height, sigma = _estimate_gaussian_parameters(signal, x1, x2,
```

Comment:
Only the `sigma` is estimated here which make it fairly inaccurate expect when the degree of gaussianity is high (in which case, it may be better to use a gaussian!).
If someone could find a more accurate way to estimate the `gamma` value, that would be great, I started to look at this in the literature, but I haven't successful so far.
Even if this is quite critical, we can leave to another PR.

Reply:
I agree, but I also have not found a good solution so far, thus I propose to merge this PR and leave it. You could open an issue if you deem it appropriate.